src/Parser/Core/Modules/Items/BFA/ZandalariLoaFigurine.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
```

Comment:
Your string enclosure is inconsistent, while consistency is most important single quotes are preferred.